### PR TITLE
Features/display location map 81

### PR DESCRIPTION
**Pull request by djarredon**
Added map object to homepage, the style of the map was agreed upon by myself, Matt, and Halala, but is open to opinions by everyone.

### CODE REVIEW

File: client/src/containers/map/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
```
Comment (Domicilius) [inline client/src/containers/map/index.js]: Comment out/remove to fix linting errors

File: client/src/containers/map/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { compose, withProps } from 'recompose'
+import {
+  withScriptjs,
+  withGoogleMap,
+  GoogleMap,
+  Marker
```
Comment (Domicilius) [inline client/src/containers/map/index.js]: Comment out/remove to fix linting errors

File: client/src/containers/home/index.js [hunk 3]
```diff
@@ -2,15 +2,14 @@ import React from 'react'
 import { push } from 'react-router-redux'
 import { bindActionCreators } from 'redux'
 import { connect } from 'react-redux'
+import Map from '../map'
 
 const Home = props => {
   const { changePage, message } = props
```
Comment (Domicilius) [inline client/src/containers/home/index.js]: Comment out until we need it to prevent linting errors

**Review by Domicilius — APPROVED** [review]
Changes look fine, only a few linting errors. Lines commented out will mostly be needed later so abstain on removing them in favor of commenting out.

File: client/src/containers/home/index.js (hunk 3 above)
Comment (djarredon) [reply client/src/containers/home/index.js]: Are you asking to comment out the `import Map` line? This doesn't create any errors for me.

**Review by iandusenbury — APPROVED** [review]

**Pull request merged**